rigex-front-end/src/styles/colors.scss

```diff
@@ -9,3 +9,4 @@ $hyperlink: #0069c4;
 $divider: #e0e5e7;
 $new-account-text: #526c7b;
 $input-field-text: #00263d;
+$shadow: grey;
```

Comment:
Is there a particular hex value for the type of grey in the designs?

Reply:
I don't think so, but will change to a hex to be consistent.